pkg/ccl/streamingccl/streamingest/alter_replication_job_test.go

```diff
@@ -26,6 +26,66 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestAlterTenantCompleteToTime(t *testing.T) {
```

Comment:
I do wonder if these tests could be data-driven tests.  But they are already written, so I leave that to your judgement.

Reply:
good question.. I think I like the data driven tests for running through a user story such as setting up clusters and writing some things and doing a cutover etc., but here these 2 tests are the minimum code needed to verify one small thing which I think is useful. we can also do that with data driven but isn't it a bit harder to debug data driven tests? anyway, my point is that we need both but I see golang tests as better for unit tests and data driven better for longer story tests.
that said, I'll definitely try to remind myself to use data driven more..